pkg/controller/controller_test.go

```diff
@@ -1843,6 +1838,28 @@ func TestCatalogConversionClusterServicePlanBindable(t *testing.T) {
 		},
 	}
 
+	preexistingServiceClasses := make(map[string]*v1beta1.ClusterServiceClass)
+	for _, eclass := range eclasses {
+		preexistingServiceClasses[eclass.Spec.ExternalID] = &v1beta1.ClusterServiceClass{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: eclass.Name,
+			},
+		}
+	}
+	preexistingServicePlans := make(map[string]*v1beta1.ClusterServicePlan)
+	for _, eplan := range eplans {
+		preexistingServicePlans[eplan.Spec.ExternalID] = &v1beta1.ClusterServicePlan{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: eplan.Name,
+			},
+		}
+	}
+
+	aclasses, aplans, err := convertAndFilterCatalog(catalog, nil, preexistingServiceClasses, preexistingServicePlans)
```

Comment:
I think all of the tests have the same value for the k8s.name and the externalID - I wonder if we should change some of them to be different to make sure we don't have code that assumes they're supposed to be the same?